### PR TITLE
Optimizations, new option, code improvements.

### PR DESCRIPTION
**Pull request by AyeItsAxi**
Replaced most if statement terror with switch statements, added extra option to prevent lightning from breaking time circuits and hover capabilities with help from nomakewan. General optimizations and cleaner code in some files.

### CODE REVIEW

**Comment by VictorGamer072YT** [comment]
rip git commit names

**Review by of-ravens-claw — CHANGES_REQUESTED** [review]
Both commit ([1e2ad42](https://github.com/nomakewan/BackToTheFutureV/pull/1/commits/1e2ad42b65bee569f10fc34658f3aed0f33f4eba)) and [2b6f3f9](https://github.com/nomakewan/BackToTheFutureV/pull/1/commits/2b6f3f9da5d6ff55548b842d95ac0af19a58c353) are useless. I suggest pulling the origin before making a PR.
Commit names might be useful to change, too.
FYI, you edited the version from `2.2.*` to `2.2.4`, whilst the commit history suggests it would be `2.3.*`, Your copy of the repo *might* be broken.

The plutonium buy limit. Just, why?

Avoid modifying the .csproj for no reason, specifically in [7d1623f](https://github.com/nomakewan/BackToTheFutureV/pull/1/commits/7d1623fd9f9aef13e484d56189377ecfdb4d6de2).

Commit [3e59c2c](https://github.com/nomakewan/BackToTheFutureV/pull/1/commits/3e59c2c0dd1e2b23587b35a9ca1eb1b878f88cc7) could be merged with [7d1623f](https://github.com/nomakewan/BackToTheFutureV/pull/1/commits/7d1623fd9f9aef13e484d56189377ecfdb4d6de2).

The actual if -> switch part looks fine, though.
But as ran said earlier, is there truly a reason to refactor this? Besides improving reading a bit, it doesn't really do much.

**Comment by of-ravens-claw** [comment]
Also, I have no clue on how GitHub's review system works, so ignore that.

File: BackToTheFutureV/Menu/GarageMenu.cs [hunk 1]
```diff
@@ -225,7 +225,7 @@ public override void Tick()
             repairEngine.Enabled = active && FusionUtils.CurrentTime.Year >= 1912 && (CurrentTimeMachine.Vehicle.EngineHealth <= 0 && CurrentTimeMachine.Mods.Wheels.Burst);
             washCar.Enabled = active && CurrentTimeMachine.Vehicle.DirtLevel > 0;
 
-            buyPlutonium.Enabled = InternalInventory.Current.Plutonium < 5 && FusionUtils.CurrentTime.Year > 1985 && FusionUtils.CurrentTime.Year < 2015;
+            buyPlutonium.Enabled = FusionUtils.CurrentTime.Year > 1985 && FusionUtils.CurrentTime.Year < 2015;
```
Comment (VictorGamer072YT) [inline BackToTheFutureV/Menu/GarageMenu.cs:211]: Just, why?

**Comment by ranstar74** [comment]
[2b6f3f9](https://github.com/nomakewan/BackToTheFutureV/pull/1/commits/2b6f3f9da5d6ff55548b842d95ac0af19a58c353) breaks relative dependency paths in project file

**Pull request closed without merging**